test/parallel/test-tls-multi-key.js

```diff
@@ -1,4 +1,4 @@
-// Copyright Joyent, Inc. and other Node contributors.
+no// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
unexpected manual error? please correct it

Reply:
Oh dear...trackpad issues! Thanks! Fixed!